toolkit/templates/forms/_checkbox-tree-selection-buttons.html

```diff
@@ -34,14 +34,7 @@ <h3 class="categories-heading">{{ parent_category.label }}</h3>
                 last=loop.last,
                 hint=None
               %}
-
-                {# TODO: PLS FIX #}
-                {% if PAGES_BUILDER %}
-                  {% include 'forms/_selection-button.html' %}
-                {% else %}
-                  {% include 'toolkit/forms/_selection-button.html' %}
-                {% endif %}
-
+                {% include 'toolkit/forms/_selection-button.html' %}
               {% endwith %}
```

Comment:
So this pull request would silently remove the 'toolkit/' part of include filepaths, but only when building pages for the toolkit itself.

ie, if you call this
```
 {% include 'toolkit/forms/_selection-button.html' %}
```
it will actually look for the path at `forms/_selection-button.html` when it's building the pages. 

It's a bit gross, but it allows us to:
- use the same path across both the frontend toolkit and the frontend apps
- not need any code changes in the frontend apps themselves

The other option would be to use relative paths, which is what George is suggesting below.